crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -0,0 +1,217 @@
+# Assignable-to relation
+
+The `is_assignable_to(S, T)` relation below checks if type `S` is assignable to type `T` (target).
+This allows us to check if a type `S` can be used in a context where a type `T` is expected
+(function arguments, variable assignments). See the [typing documentation] for a precise definition
+of this concept.
+
+```py
```

Comment:
I thought about splitting this into multiple Markdown sections, but it didn't feel like this would improve the structure a lot, and I would have to repeat these imports multiple times.

Reply:
I did have quite a few failing assertions while writing those tests, and since we report proper spans for those static assertion failures, it's actually quite easy to jump to the exact line that is failing. If you still think it makes sense to split them anyway, I'm happy to do that. I agree that it would probably look nicer (in rendered form).

Reply:
Good point about the spans... but yeah, I still think I'd weakly prefer to have it split up, for improved readability when rendered :-)